tareas/clase-5/tarea4-mio.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Clase 5 - Tarea 4</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    
+</head>
+<body>
+    <div id="lista">
+        <ol>
+            <li>1</li>
+            <li>2</li>
+            <li>3</li>
+            <li>4</li>
+            <li>5</li>
+            <li>6</li>
+            <li>7</li>
+            <li>8</li>
+            <li>9</li>
+            <li>0</li>
+        </ol>
+    </div>
+    <div id="texto">
+        <p><em id="promedio">El promedio es...</em></p>
+        <p><em id="menor">El número más pequeño es...</em></p>
+        <p><em id="mayor">El número más grande es...</em></p>
+        <p><em id="moda">El número más frecuente es...</em></p>
+
+    </div>
+
+
+    <script type="text/javascript" src='tarea4-mio.js'></script>
+</body>
+</html>
```

Comment:
Siempre que veas el simbolito este, es que te olvidaste de agregar un enter al final. Tiene que haber una línea vacía al final de cada archivo.

```suggestion
</html>

```

Reply:
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file